docs/generated/manifests/ci.json

```diff
@@ -131,15 +131,15 @@
     "file": "",
     "itemList": [
       {
-        "id": "remote-cache",
-        "name": "Use Remote Caching",
+        "id": "remote-replay",
+        "name": "Use Remote Replay",
```

Comment:
"Remote Replay" does not make any sense to me. The "marketing term" is "Nx Replay", which replays tasks that are both, cached locally and remotely. But "Remote Replay" sounds weird to me.

Also, we should be cautious to just rename it everywhere. How would people find it searching for "Remote caching", like folks coming from other monorepo solutions?